gamedata/loc_US/loc/US/strings_db.lua

```diff
@@ -2300,13 +2300,15 @@ lobui_0119="All terrain and units visible"
 lobui_0120="Victory Condition"
 lobui_0121="Determines how a victory can be achieved"
 lobui_0122="Assassination"
-lobui_0123="Game ends when commander is destroyed"
+lobui_0123="Player is defeated when the Commander is destroyed"
 lobui_0124="Supremacy"
-lobui_0125="Game ends when all structures, commanders and engineers are destroyed"
+lobui_0125="Player is defeated when all engineers, factories and any unit that can build an engineer are destroyed"
 lobui_0126="Annihilation"
-lobui_0127="Game ends when all units are destroyed"
+lobui_0127="Player is defeated when all structures (except walls) and all units are destroyed"
 lobui_0128="Sandbox"
-lobui_0129="Game never ends"
+lobui_0129="Player is never defeated"

```

Comment:
Dovetailing with the comment above, the sandbox descriptor should be something like "A player can never be defeated" or "No player can ever be defeated".

Reply:
I'll change all the VC descriptions once the other proposed changes are discussed & agreed upon.